openapi/components/schemas/Contact/ContactObject.yaml

```
@@ -72,6 +72,13 @@ properties:
     format: date
     nullable: true
     example: 1980-04-01
+  jobTitle:
+    description: The contact's job title.
+    type: string
+    pattern: '^[\w\s\-\/\pL,.#;:()'']+$'
+    maxLength: 60
```

```suggestion
    maxLength: 100
```

I am thinking about changing max length to 100. What do you think?